client/src/pages/Home.js

```diff
@@ -137,6 +141,7 @@ const Home = (props) => {
     <MainContainer className="text-center">
       <StyledIntro>
         <IntroText>
+          <StyledP>FightPandemics</StyledP>
```

Comment:
I think the design team just updated the designs ? This doesn't show on Figma

<img width="856" alt="Screen Shot 2020-05-10 at 11 57 37 PM" src="https://user-images.githubusercontent.com/42228386/81533573-476dce00-931b-11ea-9101-5272c478fb56.png">

Reply:
I'll check the new designs, they are on Figma now

Reply:
I'm a bit confused... There are two styles for the landing page on Figma, one says "FightPandemics" above the "A place..." and the other doesnt

Reply:
Yikes that's confusing... let's clarify with the design team

Reply:
I reached out on Slack